app/components/subject-viewer.cjsx

```diff
@@ -158,6 +158,26 @@ module.exports = createReactClass
                   <button aria-label="Play" title="Play" type="button" className="secret-button subject-tools__play" onClick={@setPlaying.bind this, true}>
                     <i className="fa fa-play fa-lg fa-fw"></i>
                   </button>}
+                <div className="subject-frame-duration">
+                  <button aria-label="Slower" title="Slower" type="button" className="secret-button" onClick={@setFrameDuration.bind this, -0.25}>
```

Comment:
`onMouseDown`/`onKeyDown` might be better than `onClick` here, so that I can lower the speed by holding the button down or holding enter down. The keyboard handler would have to filter keys for enter/space.